com.github.Qv2ray.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "rename-appdata-file": "qv2ray.metainfo.xml",
+    "cleanup": [
+        "/include",
+        "/lib/debug",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
```

Comment:
Why you would need host access?

Reply:
Some users will put backend(v2ray) to /usr. That's the reason why we need this.

Reply:
Like [Arch](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/v2ray)